superset-frontend/src/explore/components/DatasourcePanel/index.tsx

```diff
@@ -289,6 +302,17 @@ export default function DataSourcePanel({
     [lists.columns, showAllColumns],
   );
 
+  const showInfoboxCheck = () => {
+    if (sessionStorage.getItem('showInfobox') === 'false') return false;
+    return true;
+  };
+
+  const datasourceTypeCheck =
```

Comment:
I would maybe name this something that sounds like a boolean like `isValidDatasourceType`

Reply:
Also a good idea! I changed the naming in [`this commit`](https://github.com/apache/superset/pull/19855/commits/90c20b23a223352b9f61e472aa36f7ebf6a60c43).